index.html

```diff
@@ -820,13 +820,25 @@ <h3>JsonWebKey</h3>
               </dd>
               <dt><dfn class="lint-ignore">publicKeyJwk</dfn></dt>
               <dd>
+                <p>
 The `publicKeyJwk` property is REQUIRED, and its value MUST
 be a JSON Web Key that conforms to [[RFC7517]].
 It is RECOMMENDED that verification methods that use
 JWKs [[RFC7517]] to represent their <a>public keys</a> use the value of `kid` as
 their fragment identifier. It is RECOMMENDED that JWK `kid` values be set to
 the public key fingerprint [[RFC7638]]. See the first key in the example below
 for an instance of a public key with a compound key identifier.
+                </p>
+                <p>
+The `alg` property identifies the algorithm intended
+for use with the public key, and is
+included to prevent security issues that can arise when using the same key with multiple
```

Comment:
> [@selfissued] I misunderstood the context when we were discussing this on the call. `alg` is a required header parameter, which is what I thought we were discussing. `alg` is not required in JWKs.

Reverting incorrect changes made since https://github.com/w3c/vc-jose-cose/pull/220#pullrequestreview-1841798646, and improving the result
```suggestion
The `alg` property is OPTIONAL in JWKs, and identifies the algorithm intended
for use with the public key. Although optional, it is RECOMMENDED that `alg`
be included, to avoid security issues that can arise when using the same key with multiple
```

Reply:
@TallTed this change was intentional, as we do not want to overlap with normative guidance provided by the JWK spec

Reply:
@decentralgabe — WHICH change was intentional?

[This specific change request](https://github.com/w3c/vc-jose-cose/pull/220#pullrequestreview-1849515696) is to bring us back to where we were before @selfissued's misunderstanding of `alg` property vs `alg` header parameter, where there were no complaints about any "overlap with normative guidance provided by the JWK spec". 

If we're now (near to) overlapping the JWK spec, we should have a citation here which points to the specific segment of the JWK spec, such that implementers of `vc-jose-cose` do not have to read the entirety of the JWK spec.

Reply:
the change to remove the OPTIONAL and RECOMMENDED - as you note - before the misunderstanding - was intentional.

> If we're now (near to) overlapping the JWK spec, we should have a citation here which points to the specific segment of the JWK spec, such that implementers of `vc-jose-cose` do not have to read the entirety of the JWK spec.

This is a good idea, I will update the PR